docs/dev_coding_standard.rst

```diff
@@ -10,6 +10,20 @@ In essence we seek code that looks good, is easy to read and has great documenta
 The following is a list of exceptions/deviations from the google style guide that have been agreed upon for libCellML software development purposes.  Think of it like case-law.
 
  * lower camel case class method names
+ * indent code 4 spaces at a time (no tabs)
+
+Doxygen Comments
+================
+
+ * Doxygen comments should be sentence-style: beginning with capitalisation and ending with punctuation. However, they do not need to form grammatically correct sentences.
+
+   * An exception for capitalisation is code-words (e.g., true, false, std::string), which should be styled as typewriter text with a preceding "@c".
+ 
+Test Naming
+===========
+
+ * lower camel case 
```

Comment:
Replace with "Names should respect the lower camel case convention." or something along those lines.